Add a basic test plan

vojtapolasek opened the pull request:
Adding a TESTING.md file which lists some test cases for Vojtux.

These test cases are currently only manual.

They actually summarize cases which should be tested so that the image is as accessible as possible.

The file can also help contributors to evaluate if they did not introduce any accessibility regression.

vojtapolasek commented:
@tyrylu  I would like to hear your opinions on this.

tyrylu commented:
A few observations:
* Can not TC 2 use the shorter setup sequence from TC 3 and friends?
* The prerequisite in the setup steps of TC 4 is somewhat vague in my opinion - how do you ensure that? Especially without sight, I guess.
* For TC 5 - is pandoc installed in the live image (not tested or looked, just that it should)?
* Do we want test cases for BRLTTY (Orca can talk to it, it works on a non-graphical TTY)?

vojtapolasek commented:
> A few observations:
> 
>     * Can not TC 2 use the shorter setup sequence from TC 3 and friends?
> 
Implemented, thanks.

>     * The prerequisite in the setup steps of TC 4 is somewhat vague in my opinion - how do you ensure that? Especially without sight, I guess.
> 
Makes sense, I changed it.

>     * For TC 5 - is pandoc installed in the live image (not tested or looked, just that it should)?
> 
It was not installed, but I added it.

>     * Do we want test cases for BRLTTY (Orca can talk to it, it works on a non-graphical TTY)?

I will try to add some. I don't have braille display with me right now, so I will add it later.
By the way, is there any virtual brltty display so that even people without the device can verify this? Do you know how to set it up?

tyrylu commented:
No idea about the virtual display, likely ask Bohdan.

vojtapolasek commented:
@tyrylu  added test cases for brltty. I did not manage to get the virtual braille display working so far, I will ad those tests later. Any other suggestions?

vojtapolasek commented:
So after some advices, I managed to get brltty-xw to work and I added information to the plan. I am merging this. Thanks everyone for advices.

ggbecker commented on TESTING.md:
```diff
@@ -0,0 +1,489 @@
+
+- Press ALT+SUPER+L.
+
+### Expected results:
+
+- Press ALT+SUPER+L.
+
```
Duplicate

vojtapolasek replied:
fixed, thank you.
